xbmc/video/VideoInfoTag.cpp

```diff
@@ -320,7 +333,12 @@ void CVideoInfoTag::Archive(CArchive& ar)
     ar << m_iSeason;
     ar << m_iEpisode;
     ar << m_strUniqueId;
-    ar << m_fRating;
+    RatingMap::iterator i;
```

Comment:
You first need to store the number of rating items that exist and then serialize the ratings (with all their details).